.drone.yml

```diff
@@ -7,52 +7,114 @@ platform:
   os: linux
   arch: amd64
 
+resources:
+  requests:
+    cpu: 1000
+    memory: 1024Mi
+
+environment:
+  OCI_IMAGE_REPO: quay.io/ukhomeofficedigital/alpine
+
 steps:
 - name: build
-  pull: if-not-exists
+  pull: Always
   image: 340268328991.dkr.ecr.eu-west-2.amazonaws.com/acp/dind
+  resources:
+    limits:
+      cpu: 1000
+      memory: 1024Mi
   commands:
   # wait for docker service to be up before running docker build
-  - n=0; while [ "$n" -lt 60 ] && ! docker stats --no-stream; do n=$(( n + 1 )); sleep 1; done
-  - if ! docker stats --no-stream; then echo "Docker appears not to be running"; exit 1; fi
-  - docker build -t alpine:$${DRONE_COMMIT_SHA} . --no-cache
-  resources: &minimal
-    requests:
-      cpu: 500
-      memory: 512Mi
+  - n=0; while [ "$n" -lt 60 ] && [ ! "$(docker stats --no-stream)" ]; do n=$(( n + 1 )); sleep 1; done
```

Comment:
This can be replaced with `/usr/local/bin/wait`. This while loop is no longer recommended.

Reply:
This has now been done